guides/common/modules/proc_configuring-the-freeipa-server-to-use-cross-forest-trust.adoc

```diff
@@ -14,9 +14,17 @@ For example:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-[nss]
```

Comment:
The procedure description above talks about the `nss` and `domain` secrtions, but you remove the `nss` section.